packages/base-ui/src/services/confirm/confirm.service.ts

```diff
@@ -0,0 +1,14 @@
+import { createIdentifier, IDisposable } from '@wendellhu/redi';
+import { type Subject } from 'rxjs';
+
+import { type IConfirmPartMethodOptions } from '../../views/components/confirm-part/interface';
+
+export const IConfirmService = createIdentifier<IConfirmService>('univer.confirm-service');
+
+export interface IConfirmService {
+    open(params: IConfirmPartMethodOptions): IDisposable;
+
+    close(): void;
```

Comment:
Is it more appropriate adding a parameter here to close a specific confirm dialog?